crates/cxx-qt-lib/Cargo.toml

```diff
@@ -11,13 +11,11 @@ edition.workspace = true
 license.workspace = true
 description = "Qt types for integrating `cxx-qt` crate with `cxx`"
 repository.workspace = true
-# Fake linking so that we expose cargo data as env vars for downstream crates
-# https://github.com/rust-lang/cargo/issues/3544
-links = "cxx-qt-lib"
```

Comment:
i was hoping removing this might help the Windows issue, but appears not. Something else is broken ...

Reply:
All we have to go on is the following which suggests some dll is missing at some point in the qml-feature build script. But it's unclear if it's a Qt one or a system one or a cxx-qt one or if it's when building the build script or building the final binary.

`  process didn't exit successfully: `C:/cxx-qt/build/./cargo/build\release\build\qml-features-814177af2461bb34\build-script-build` (exit code: 0xc0000135, STATUS_DLL_NOT_FOUND)`

Reply:
Appears when you have `cxx-qt-lib` as a build dependency the issue occurs so keeping the cxx-qt-lib-headers split works for now...